com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ArchiveMerger.java

```diff
@@ -52,8 +48,11 @@ public void createArchiveFromParts(List<FileEntry> archivePartEntries) {
             stepLogger.info(Messages.ERROR_MERGING_ARCHIVE);
             throw new SLException(e, Messages.ERROR_MERGING_ARCHIVE_PARTS, e.getMessage());
         } finally {
-            deleteMergedFile(archiveMerger);
```

Comment:
This file will be deleted in ValidateDeployParametersStep. I moved it there because I need to use the archive (saved on the container) to execute the verifications.